upgrade/upd_2.5.11-to-2.7.0/index.php

```
@@ -208,75 +250,507 @@ public function check_addsessioncookieprefs()
+            $maintenance = new \SystemMaintenance();
+            $result = $maintenance->CleanCache([1, 2, 3]);
+            if (true === $result) {
+                $_SESSION[$this->cleanCacheKey] = true;
+            }
+            return $result;
+        } catch (\Throwable $e) {
+            $this->logs[] = 'Failed to clean cache: ' . $this->sanitizeLogMessage($e->getMessage());
+
+            return false;
+        }
+    }
```

PHPMD (2.15.0)

[error] apply_cleancache accesses the super-global variable $_SESSION. (undefined)

(Superglobals)